### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### LINKED ISSUE #21: Create a PR template
Create a new PR template which should be used when raising PRs.

### PR DESCRIPTION
**Pull request by 098suraj**
Closes #21

### CODE REVIEW

**Review by chinmoy12c — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-#### Why is this the best possible solution? Were any other approaches considered?
+List the changes done to fix a bug or introduce a new feature.
 
-#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
+## Checklist
 
-#### Do we need any specific form for testing your changes? If so, please attach one.
+Please check the boxes below that apply:
 
-#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.
```
Comment (chinmoy12c) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: This statement is not required, the checklist below should suffice.

**Review by chinmoy12c — CHANGES_REQUESTED** [review]

**Comment by chinmoy12c** [comment]
Thanks for the PR @098suraj, please take note that future PRs should be based on `Samagra-Development/odk-collect-extension`'s `develop` branch and not on `getodk/collect`.

**Review by karntrehan — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (098suraj) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: The line was there in the original PR Template. I deleted it in here. The original template dosen't contain this. I have attached the link to the original one below.

https://github.com/getodk/collect/blob/9c6b34833c2c9b2ff3cc063e39bb110ea0e16663/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L18

**Comment by 098suraj** [comment]
> Thanks for the PR @098suraj, please take note that future PRs should be based on `Samagra-Development/odk-collect-extension`'s `develop` branch and not on `getodk/collect`.

My bad! I failed to notice it this time. Will take care of it from the next time. Thanks @chinmoy12c for pointing this out :-)

**Comment by chinmoy12c** [comment]
Hey, @098suraj can you please rebase from develop and force-push your branch once?

**Comment by 098suraj** [comment]
Hey @chinmoy12c! Rebasing led to some conflicts and unnecessary merge commits. I created a new pull request #37 based on the `develop` branch to keep the commit history clean. Also, I'll remember to base my next contributions on `develop` branch and maintain a clean commit history. Closing this one for now.

**Pull request closed without merging**